regtest.sh

```diff
@@ -240,6 +240,8 @@ make-summary() {
     | sed -e '/TABLE_ROWS/ r rows.html' \
     > $filename
 
+  rm rows.html
```

Comment:
"./test.sh all" did not flag anything, so I merged the branch into master
and deleted it.

On Wed, May 27, 2015 at 12:59 PM andychu <notifications@github.com> wrote:

> In regtest.sh
> <https://github.com/google/rappor/pull/37#discussion_r31173817>:
>
> > @@ -240,6 +240,8 @@ make-summary() {
> >      | sed -e '/TABLE_ROWS/ r rows.html' \
> >      > $filename
> >
> > +  rm rows.html
>
> LGTM
> On Tue, May 26, 2015 at 8:50 PM, Ilya Mironov <notifications@github.com>
> wrote: In regtest.sh <
> https://github.com/google/rappor/pull/37#discussion_r31101404>: > @@
> -240,6 +240,8 @@ make-summary() { > | sed -e '/TABLE_ROWS/ r rows.html' \ >
> > $filename > > + rm rows.html I removed the line for now, but still wonder
> - why do we need the rows.html file lying around?
> It's an intermediate output, and we so far we keep all intermediate
> outputs. We could move them if they're in the way but I think it's nice to
> have for debuggability. Make sure './test.sh all' passes (I think I only
> checked ./test.sh r-unit) and then go ahead and submit. This may catch some
> lint errors, etc. thanks, Andy
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/rappor/pull/37/files#r31173817>.
>